templates/engage/ubuntu-tech-live-episode-2.html

```diff
@@ -0,0 +1,140 @@
+{% extends "engage/base_engage.html" %}
+
+{% block title %}Ubuntu Tech Live第二期{% endblock %}
+{% block meta_description %}Ubuntu Tech Live第二期Ubuntu Core 20新版发布{% endblock %}
+
+{% block content %}
+
+<section class="p-strip is-deep u-no-padding--top u-no-padding--bottom">
+  <div class="p-strip--suru-topped">
+    <div class="row u-vertically-center">
+      <div class="col-6">
+        <h1>Ubuntu Tech Live第二期：Ubuntu Core 20新版本抢先新体验</h1>
+          <p class="p-heading--four">想了解Ubuntu Core 20最新特性？注册并观看研讨会视频直播就有机会获得官方纪念品</p>
+      </div>
+      <div class="u-vertically-center col-5 col-start-large-8 u-align--center u-hide--small">
+      <div class=" lazyloaded" data-noscript="">
```

Comment:
You don't need this div. This is auto generated on Ubuntu.com

Reply:
Updated